issues/getTouchedAnnotation.ts

```diff
@@ -35,31 +77,49 @@ export default function getTouchedAnnotation(
     const top = annot.top;
     const right = left + annot.width;
     const bottom = top + annot.height;
-    const annotCenterX = (right - left) / 2 + left;
-    const annotCenterY = (bottom - top) / 2 + top;
     const isWithinBounds =
       left <= point.x &&
       right >= point.x &&
       top <= point.y &&
       bottom >= point.y;
-
-    const deltaX = Math.abs(annotCenterX - point.x);
-    const deltaY = Math.abs(annotCenterY - point.y);
-    const distance = Math.sqrt(Math.pow(deltaX, 2) + Math.pow(deltaY, 2));
+    let distance = 0;
+  
 
     // Exact match
     if (isWithinBounds) {
       nearestDistance = 0;
       match = annot;
       return;
     }
+    //perpendicular match on y axis
```

Comment:
Nice thinking (looking at your notes) 😁 The solution is not perhaps the most elegant one but seems to do it's job.